processor.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+var doiuse = require('doiuse');
+var github = require('octonode');
+var postcss = require('postcss');
+var sourceMap = require('source-map');
+var stylus = require('stylus');
+var logger = require('./logger');
+
+function processCSS(originRepo, originBranch, file, config, handleIncompatibility) {
+    var repoClient = github.client().repo(originRepo);
+
+    repoClient.contents(file.filename, originBranch, function(error, fileContentsMetadata) {
+        var fileContents;
+
+        if (error) return logger.error('Error reading CSS file contents:', error);
+
+        fileContents = new Buffer(fileContentsMetadata.content, 'base64').toString();
+
+        postcss(doiuse({
+            browsers: config,
+            onFeatureUsage: handleIncompatibility
+        })).process(fileContents, {
+            from: '/' + file.filename
+        }).then(function(response) {});
```

Comment:
I think we should eliminate this line.  I like that we put the `then` there as a reminder we *can* do something here, but since we aren't, we should axe for now.